api/main.go

```diff
@@ -95,6 +95,11 @@ func NewRunner(cfg *configuration.APIRunner) (*Runner, error) {
 		return nil, err
 	}
 
+	err = rpcServer.RegisterService(NewRegisterNodeService(&ar), "register_node")
```

Comment:
rename all of this to receive_node_cert or similar. Its cant be named "register"  because NodeRecord already created